### PR TITLE
Update files for 5.1.1.3 release

### PR DESCRIPTION
**Pull request by lostasi**
- `base` directory is now the `config` directory
- Cluster sample files have been relocated to `generated` folder
- `machineconfig` folder has been relocated under the `config` directory

### CODE REVIEW

File: config/scale/cluster/samples/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+commonLabels:
+  app.kubernetes.io/instance: ibm-spectrum-scale
+  app.kubernetes.io/name: cluster
+
+resources:
+  - ../bases
+  - ../cluster
+
+patchesStrategicMerge:
+  - delete_filesystem.yaml
```
Comment (whowutwut) [inline config/scale/cluster/samples/kustomization.yaml:12]: need to delete this line, looks like we removed the vdisknsd-sample, kustomize build fails here.

File: config/scale/cluster/samples/delete_filesystem.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# delete the ECE filesystem pieces
+---
+$patch: delete
+apiVersion: scale.spectrum.ibm.com/v1beta1
+kind: Filesystem
+metadata:
+  name: vdisknsd-sample
```
Comment (whowutwut) [inline config/scale/cluster/samples/delete_filesystem.yaml:7]: And also delete this file completely, since it doesn't apply.

**Review by whowutwut — CHANGES_REQUESTED** [review]
Only this comment about removing the delete filesystem related stuff, so kustomize will run successfully.

**Comment by whowutwut** [comment]
@lostasi i assumed that we removed all the directories in the old main repo, and then copied the new files over with a fresh start right? In that case, i didn't review the contents of the files as carefully as we have deployed it and it should work.

File: config/scale/cluster/filesystem/kustomization.yaml [hunk 3]
```diff
@@ -5,6 +5,7 @@ commonLabels:
   app.kubernetes.io/name: cluster
 
 resources:
+  - scale_v1beta1_filesystem_vdisknsd.yaml
```
Comment (whowutwut) [inline config/scale/cluster/filesystem/kustomization.yaml:8]: hummm.... looks like i missed this cleanup

**Comment by lostasi** [comment]
> @lostasi i assumed that we removed all the directories in the old main repo, and then copied the new files over with a fresh start right? In that case, i didn't review the contents of the files as carefully as we have deployed it and it should work.

I didn't do a whole removal. I did some reshuffling of where the directories live as they do for 5.1.1.3, as opposed to 5.1.1.1 structure. Should map over pretty easily I think. The README here on the site needs to stay the same.

File: config/scale/cluster/samples/kustomization.yaml (hunk 1 above)
Comment (lostasi) [reply config/scale/cluster/samples/kustomization.yaml]: Thanks for addressing this!

File: config/scale/cluster/samples/delete_filesystem.yaml (hunk 2 above)
Comment (lostasi) [reply config/scale/cluster/samples/delete_filesystem.yaml]: Thanks for addressing this!

File: config/scale/cluster/filesystem/kustomization.yaml (hunk 3 above)
Comment (lostasi) [reply config/scale/cluster/filesystem/kustomization.yaml]: You got it.

**Review by whowutwut — APPROVED** [review]

**Comment by whowutwut** [comment]
> The README here on the site needs to stay the same.


Yea, looks like no changes to the README in the files list.

**Pull request merged**